dbscan/dbscan.go

```diff
@@ -171,7 +178,22 @@ func (api *API) ScanAll(dst interface{}, rows Rows) error {
 // and propagates any errors that could pop up.
 // It scans data from that single row into the destination.
 func (api *API) ScanOne(dst interface{}, rows Rows) error {
-	return api.processRows(dst, rows, false /* multipleRows. */)
+	return api.processRows(dst, rows, false /* multipleRows. */, false /* close. */)
```

Comment:
Why do you use "false" for the `close` parameter here? I believe it should be the same as in the `ScanAll` function above, "true".

Reply:
Of course.... miss by me